tools/docker/compose

```diff
@@ -119,12 +119,14 @@ case "$1" in
     $deps up --exit-code-from wait-db wait-db
     $deps up --exit-code-from wait-db-2 wait-db-2
     set +e
```

Comment:
Wouldn't it be better not to use `set +e` ?

Reply:
I think `set +e` is the default -- we want `set -e`, don't we?

Reply:
Yeah, should be okay to error out if any of those parts fail.

Reply:
default behavior is `set +e` .. we use a `set -e` at the top of the script because we want to error ASAP if anything goes wrong in the setup / tear down or really any other part of the script. This is the only place where we expect a 1 exit code might occur but don't want to exit immediately, so we wrap it in a
```
set +e
...
set -e
```

Reply:
Small nit, but it's more idiomatic and less error-prone to suffix the command with `|| /bin/true`.

Reply:
Ah, this makes sense -- otherwise we'd lose the test logs

Reply:
Correct @spooktheducks